app/src/main/kotlin/com/wire/android/ui/common/textfield/WireTextField.kt

```diff
@@ -261,14 +264,18 @@ private fun InnerText(
                 top = 2.dp, bottom = 2.dp
             )
             if (value.text.isEmpty() && placeholderText != null) {
-                Text(
-                    text = placeholderText,
-                    style = placeholderTextStyle,
-                    color = colors.placeholderColor(state).value,
+                Column(
```

Comment:
why column?

Reply:
I wanted to add parameter to allow only for `Alignment.Horizontal`, but you motivated me to remove this `Column` to have less nesting and create a mapper from `Alignment.Horizontal` to `Alignment` 😄